src/componentsNext/Breadcrumb/ExplorerBreadcrumb/BreadcrumbSegment.tsx

```diff
@@ -18,15 +18,14 @@ const BreadcrumbSegment: FC<{
    * we need to request file information to figure out which
    * icon to use
    */
-  const requestIconInformation = isLast;
 
   const { data } = useNodeContent({
     path: absolute,
-    enabled: requestIconInformation,
+    enabled: isLast,
```

Comment:
It's easier to follow along without introducing an extra variable (and an extra thought to keep in mind).